extensions/wikia/GameGuides/css/GameGuides.scss

```diff
@@ -65,7 +53,7 @@ html {
 	}
 
 	&.serif {
-		font-family: Palatino, "GGSerif", "Palatino Linotype", "Palatino LT STD", "Book Antiqua", Georgia, serif;
+		font-family: Palatino, "Palatino Linotype", "Palatino LT STD", "Book Antiqua", Georgia, serif;
```

Comment:
We should make sure that `Georgia, serif` fonts are acceptable by Product cause on Android `Palatino` fonts are not available by default.

Reply:
Sure, pinged Ania and Stanley in the ticket.